BB-SC/MWLayer/Power-Management/BB_Power_Management.md

```diff
@@ -0,0 +1,84 @@
+
+# Power Management
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Power Management
+
+## Layer
+<!-- 1, 2a, 2b, 3 -->
+<!-- NOTE: Flag -->
+MWLayer
```

Comment:
Isn't this an OSLayer concern?

Reply:
Agreed. Needs discussion in WP2.

Reply:
rename to "Power_Management_Coordination"

Reply:
done